exporter/pulsarexporter/README.md

```diff
@@ -0,0 +1,66 @@
+# Pulsar Exporter
+
```

Comment:
Please add a standard doc header with "in development" status. Check how it's done for other components

Reply:
RESOLVED, please help approve test again. @dmitryax